### ISSUE TITLE
Inclusion of tests makes the package very heavy

### ISSUE THREAD

**Issue by spicymatt** [hughbe/PhotoshopReader#1]
hello
I am using SPM to access your framework in xCode. I regret that this package is soooo big (its one the heaviest I am using). IS there a way to avoid downloading all those test images?

Matthieu

**Comment by hughbe** [comment]
Good point, but I don't really know how - is there a way in swiftpm not to include tests?
